engine/src/main/resources/org/hibernate/validator/ValidationMessages_zh_CN.properties

```diff
@@ -1,27 +1,43 @@
-org.hibernate.validator.constraints.Email.message            = \u4E0D\u662F\u4E00\u4E2A\u5408\u6CD5\u7684\u7535\u5B50\u90AE\u4EF6\u5730\u5740
-org.hibernate.validator.constraints.Length.message           = \u957F\u5EA6\u9700\u8981\u5728{min}\u548C{max}\u4E4B\u95F4
-org.hibernate.validator.constraints.CodePointLength.message  = \u957F\u5EA6\u9700\u8981\u5728{min}\u548C{max}\u4E4B\u95F4
-org.hibernate.validator.constraints.NotBlank.message         = \u4E0D\u80FD\u4E3A\u7A7A
-org.hibernate.validator.constraints.NotEmpty.message         = \u4E0D\u80FD\u4E3A\u7A7A
-org.hibernate.validator.constraints.Range.message            = \u9700\u8981\u5728{min}\u548C{max}\u4E4B\u95F4
-org.hibernate.validator.constraints.SafeHtml.message         = \u53EF\u80FD\u6709\u4E0D\u5B89\u5168\u7684HTML\u5185\u5BB9
-org.hibernate.validator.constraints.ScriptAssert.message     = \u6267\u884C\u811A\u672C\u8868\u8FBE\u5F0F"{script}"\u6CA1\u6709\u80FD\u591F\u5F97\u5230true
-org.hibernate.validator.constraints.URL.message              = \u9700\u8981\u662F\u4E00\u4E2A\u5408\u6CD5\u7684URL
+org.hibernate.validator.constraints.CreditCardNumber.message        = \u4e0d\u5408\u6cd5\u7684\u4fe1\u7528\u5361\u53f7\u7801
+org.hibernate.validator.constraints.Currency.message                = \u4e0d\u5408\u6cd5\u7684\u8d27\u5e01 (\u5fc5\u987b\u662f{value}\u5176\u4e2d\u4e4b\u4e00)
+org.hibernate.validator.constraints.EAN.message                     = \u4e0d\u5408\u6cd5\u7684{type}\u6761\u5f62\u7801
+org.hibernate.validator.constraints.Email.message                   = \u4e0d\u662f\u4e00\u4e2a\u5408\u6cd5\u7684\u7535\u5b50\u90ae\u4ef6\u5730\u5740
+org.hibernate.validator.constraints.Length.message                  = \u957f\u5ea6\u9700\u8981\u5728{min}\u548c{max}\u4e4b\u95f4
+org.hibernate.validator.constraints.CodePointLength.message         = \u957f\u5ea6\u9700\u8981\u5728{min}\u548c{max}\u4e4b\u95f4
+org.hibernate.validator.constraints.LuhnCheck.message               = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, Luhn\u6a2110\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.Mod10Check.message              = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, \u6a2110\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.Mod11Check.message              = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, \u6a2111\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.ModCheck.message                = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, ${modType}\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.NotBlank.message                = \u4e0d\u80fd\u4e3a\u7a7a
+org.hibernate.validator.constraints.NotEmpty.message                = \u4e0d\u80fd\u4e3a\u7a7a
+org.hibernate.validator.constraints.ParametersScriptAssert.message  = script expression "{script}" didn't evaluate to true
```

Comment:
Hi @quaff ,

Thanks for the update. Any reason why you didn't translate the line just above?

Reply:
Sorry, I forgot it, I have submitted another commit.